app/controllers/api/v1/impersonates_controller.rb

```diff
@@ -2,13 +2,17 @@
 
 module API
   module V1
-    class UsersController < API::V1::APIController
+    class ImpersonatesController < API::V1::APIController
```

Comment:
I think we are using `Impersonation` in other classes, does it make sense to name it Impersonation here?